routers/api/v1/org/action.go

```diff
@@ -151,6 +151,10 @@ func DeleteOrgSecret(ctx *context.APIContext) {
 	//   "403":
 	//     "$ref": "#/responses/forbidden"
 	secretName := ctx.Params(":secretname")
+	if err := actions.NameRegexMatch(secretName); err != nil {
```

Comment:
Do we need that check when deleting a secret? If the name is invalid, the secret should not exists, so the not found error will be returned.

Reply:
Let's check the input type before executing the model layer.

Reply:
I agree with KN4CK3R , since the result is the same, this check is a no-op.

I can't understand what's the meaning of "Let's check the input type before executing the model layer."

Do you mean "this is right and I don't want to change", or do you mean "I will make some other changes"?

Reply:
I think the intent is to not bother the database if there could be no result.

Reply:
I personally believe that all parameter validation should be done within the API layer, rather than validating parameters at the DB layer.